src/View/Parsers/HTMLValue.php

```diff
@@ -32,7 +32,7 @@ public function __construct($fragment = null)
      */
     public function setContent($content)
     {
-        $content = preg_replace('#</?(html|head|body)[^>]*>#si', '', $content);
+        $content = preg_replace('#</?(html|head|body)(\s[^>]*)?>#si', '', $content);
```

Comment:
Went for the more broad approach here rather than the suggestion in the issue - HTML6 could introduce new tags like `<heading>`, or someone could be using a front-end JS library that consumes custom tags that this would otherwise strip.

Reply:
UPDATE: I've reverted back to using the suggestion from the issue because it doesn't have the same CI failure my attempt did. Specifically there was an edge case (which we already have a unit test for, which is what failed) where some really weird invalid HTML could result in multiple `<body>` tags existing in the HTML.

The `HTMLValue` implementation relies on exactly one `<body>` tag being present, so that's a non-starter. People who want custom weird tags will have to name them something else, and we'll just have to be aware of changes when future versions of HTML roll around.